src/util/PedModels.hpp

```diff
@@ -0,0 +1,14 @@
eMule_01", "unknown"}, {"A_C_HorseMulePainted_01", "unknown"}, {"A_C_Iguana_01", "unknown"}, {"A_C_IguanaDesert_01", "unknown"}, {"A_C_Javelina_01", "unknown"}, {"A_C_LionMangy_01", "unknown"}, {"A_C_Loon_01", "unknown"}, {"A_C_Moose_01", "Male"}, {"A_C_Muskrat_01", "unknown"}, {"A_C_Oriole_01", "unknown"}, {"A_C_Owl_01", "unknown"}, {"A_C_Ox_01", "unknown"}, {"A_C_Panther_01", "unknown"}, {"A_C_PantherLegendaryr_01", "unknown"}, {"A_C_Parrot_01", "unknown"}, {"A_C_Pelican_01", "unknown"}, {"A_C_Pheasant_01", "unknown"}, {"A_C_Pig_01", "Male"}, {"A_C_Pigeon", "unknown"}, {"A_C_Possum_01", "unknown"}, {"A_C_PrairieChicken_01", "unknown"}, {"A_C_Pronghorn_01", "unknown"}, {"A_C_Quail_01", "unknown"}, {"A_C_Rabbit_01", "unknown"}, {"A_C_Raccoon_01", "unknown"}, {"A_C_Rat_01", "unknown"}, {"A_C_Raven_01", "unknown"}, {"A_C_RedFootedBooby_01", "unknown"}, {"A_C_Robin_01", "unknown"}, {"A_C_Rooster_01", "unknown"}, {"A_C_RoseateSpoonbill_01", "unknown"}, {"A_C_Seagull_01", "unknown"}, {"A_C_SharkHammerhead_01", "unknown"}, {"A_C_SharkTiger", "unknown"}, {"A_C_Sheep_01", "Male"}, {"A_C_Skunk_01", "unknown"}, {"A_C_Snake_01", "unknown"}, {"A_C_Snake_Pelt_01", "unknown"}, {"A_C_SnakeBlackTailRattle_01", "unknown"}, {"A_C_SnakeBlackTailRattle_Pelt_01", "unknown"}, {"A_C_SnakeFerDeLance_01", "unknown"}, {"A_C_SnakeFerDeLance_Pelt_01", "unknown"}, {"A_C_SnakeRedBoa_01", "unknown"}, {"A_C_SnakeRedBoa_Pelt_01", "unknown"}, {"A_C_SnakeRedBoa10ft_01", "unknown"}, {"A_C_SnakeWater_01", "unknown"}, {"A_C_SnakeWater_Pelt_01", "unknown"}, {"A_C_SongBird_01", "unknown"}, {"A_C_Sparrow_01", "unknown"}, {"A_C_Spot_Gang_Charles", "Male"}, {"A_C_Squirrel_01", "unknown"}, {"A_C_Toad_01", "unknown"}, {"A_C_Turkey_01", "unknown"}, {"A_C_Turkey_02", "unknown"}, {"A_C_TurkeyWild_01", "unknown"}, {"A_C_TurtleSea_01", "unknown"}, {"A_C_TurtleSnapping_01", "unknown"}, {"A_C_Vulture_01", "unknown"}, {"A_C_Wolf", "Male"}, {"A_C_Wolf_Medium", "unknown"}, {"A_C_Wolf_Small", "unknown"}, {"A_C_Woodpecker_01", "unknown"}, {"A_C_Woodpecker_02", "unknown"}, {"armadillo", "Female"}, {"ggrcez", "Male"}, {"LegendaryPanther", "unknown"}, {"MP_A_C_Alligator_01", "unknown"}, {"MP_A_C_Beaver_01", "Male"}, {"MP_A_C_BIGHORNRAM_01", "unknown"}, {"MP_A_C_Boar_01", "unknown"}, {"MP_A_C_BUCK_01", "unknown"}, {"MP_A_C_BUFFALO_01", "unknown"}, {"MP_A_C_Cougar_01", "unknown"}, {"MP_A_C_Coyote_01", "Male"}, {"MP_A_C_DEER", "unknown"}, {"MP_A_C_DOGAMERICANFOXHOUND_01", "unknown"}, {"MP_A_C_ELK_01", "unknown"}, {"MP_A_C_FOX_01", "unknown"}, {"MP_A_C_HORSECORPSE_01", "unknown"}, {"MP_A_C_MOOSE_01", "Male"}, {"MP_A_C_OWL_01", "unknown"}, {"MP_A_C_Panther_01", "Male"}, {"MP_A_C_POSSUM_01", "unknown"}, {"MP_A_C_PRONGHORN_01", "unknown"}, {"MP_A_C_RABBIT_01", "unknown"}, {"MP_A_C_RAM_01", "unknown"}, {"MP_A_C_SHEEP_01", "unknown"}, {"MP_A_C_Wolf_01", "unknown"}, {"MP_Horse_Owlhootvictim_01", "unknown"}, {"MP_U_M_M_LEGENDARYBOUNTY_09", "Female"}, {"P_C_Horse_01", "unknown"}, {"U_F_M_RhdNudeWoman_01", "female"}};
```

Comment:
Not a big fan of embedding such a large structure when we could use maybe a json file. But seen as we don't really have data fetching like on Gta, I might let this slide for now.

Reply:
Yes, as I know of there are no methods to fetch data like there is in gta.